### ISSUE TITLE
OoT64 decomp licensing

### ISSUE THREAD

**Issue by Dragorn421** [Dragorn421/oot-licensing#1]
Hi!

You are mentioned here because you previously contributed to the [zeldaret/oot repo](https://github.com/zeldaret/oot), a decompilation of the game Ocarina of Time 64.

There is a move to license the source of that repo under [CC0](https://creativecommons.org/publicdomain/zero/1.0/), and for that we need your permission, as past contributors.

For more information: https://github.com/Dragorn421/oot-licensing/blob/main/README.md

- If you don't care, the best choice to do for the project and users of the project (researchers, fans, modders) is to agree.
- If you agree, you can reply to this issue with some message like "I agree to share all my past contributions to the oot decomp under CC0" (please include MM in the statement if you also helped with MM decomp)
- If you disagree, please get in touch! Through [github issues](https://github.com/Dragorn421/oot-licensing/issues) or [discord](https://discord.gg/rDvMsgnU2m)

**Comment by Dragorn421** [comment]
@Roman971 @fig02 @Dragorn421 @ethteck @petrie911 @Thar0 @engineer124 @louist103 @EllipticEllipsis @krimtonz @AngheloAlf @Random06457 @shawlucas @mzxrules @gamestabled @Synray @zelda2774 @Zelllll @hensldm @i82orbom @zelda2773 @Jack-Walker @tom-overton @AdamKiddle @jpburnett @glankk @Sirius902 @pixel-stuck @ProjectRevoTPP @maekclena @Kelebek1 @emilybrooks @blackgamma7 @JoshDuMan @NEstelami @nfaltermeier @kyleburnette @playerskel @Kenix3 @luryus

**Comment by Dragorn421** [comment]
@krm01 @rogual @FryJay @Faschz @MegaMech @hatal175 @notyourav @CharlesAverill @Yanis42 @jacobly0 @zbanks @vijfhoek @s5bug @soulwa @MNGoldenEagle @liamnobel @Emill @silv3rwing07 @mnzlmstr @Peppers1612 @kroemker @Half-asian @snablu @ariahiro64 @HylianFreddy @mracsys @milotier @mrneo240 @kemenaran @philtyl @srmeier @gladiac @StardustSauce @DefinedBehavior @teaguecl @arthurtilly @Electrk @andrewzwicky @CrookedPoe @L-P @cmon-meow

**Comment by Dragorn421** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by CharlesAverill** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by rogual** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by andrewzwicky** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by hatal175** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by jacobly0** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by i82orbom** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by mnzlmstr** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0Von meinem iPhone gesendetAm 03.01.2023 um 10:41 schrieb Dragorn421 ***@***.***>:﻿
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by glankk** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by ethteck** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by nfaltermeier** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by vijfhoek** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by kemenaran** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by blackgamma7** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by Electrk** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by emilybrooks** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by AngheloAlf** [comment]
I agree to share all my contributions to the OoT and MM decomp under CC0

**Comment by luryus** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by louist103** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by mracsys** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0.

**Comment by Yanis42** [comment]
I agree to share everything I worked on (and I will always be glad to)

**Comment by tom-overton** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by HylianFreddy** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by pixel-stuck** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by liamnobel** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Half-asian** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Sirius902** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by kroemker** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by gamestabled** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by NEstelami** [comment]
I agree to share all my contributions to the OoT decomp under CC0.

**Comment by Jack-Walker** [comment]
I agree to share all my contributions to the OoT decomp under CC0.

**Comment by FryJay** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by StardustSauce** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by ProjectRevoTPP** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by notyourav** [comment]
I agree to share all my contributions to oot decomp under cc0.

**Comment by gladiac** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Emill** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by s5bug** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by srmeier** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by shawlucas** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Random06457** [comment]
I agree to share all my contributions to the oot and mm decomp under CC0

**Comment by teaguecl** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by zbanks** [comment]
I agree to share all my contributions to the OOT decomp under CC0.
I agree to share all my contributions to the MM decomp under CC0.

**Comment by zelda2773** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by fig02** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by petrie911** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by krm01** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by Thar0** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by cmon-meow** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by arthurtilly** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by DefinedBehavior** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by soulwa** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Faschz** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by AdamKiddle** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by MegaMech** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by ariahiro64** [comment]
I agree to have all my contributions shared under CC0

**Comment by hensldm** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by Synray** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by JoshDuMan** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by kyleburnette** [comment]
I agree to share all my contributions to the oot and MM decomps under CC0

**Comment by Kelebek1** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by engineer124** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by maekclena** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by milotier** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by krimtonz** [comment]
I agree to share all my contributions to the oot decomp under CC0

**Comment by Kenix3** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by quaantuum** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by playerskel** [comment]
I agree to share all my contributions to the oot decomp under CC0.

**Comment by comportase** [comment]
Have @jpburnett, @EllipticEllipsis, @Roman971, @mzxrules, @CrookedPoe, @pixel-stuck, @Zelllll and others agreed to license yet? Projects are being held back by this delay.

**Comment by ethteck** [comment]
@comportase please do not single out individuals who may not have agreed. Dragorn is on top of this effort, and any outside pressure is at best counter-productive and at worst harassment. If you have questions about the process, you can reach out to Dragorn.

**Comment by Yanis42** [comment]
This is incredibly disrespectful.

**Comment by Roman971** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0, for the purpose of waiving any rights I might have over them.

**Comment by wedarobi** [comment]
Hi, I agree to share all my contributions to the oot decomp under CC0! :)

**Comment by ghost** [comment]
Just a reminder that probably not everyone will agree to do this...
